console/src/i18n/locales.ts

```diff
@@ -78,6 +78,18 @@ const runtime = {
         en: 'View',
         zh: '查看',
     },
+    'build image': {
+        en: 'Build Image',
+        zh: '构建镜像',
```

Comment:
Duplicate locale with runtime.image.build.button, remove this?

Reply:
fixed